### PR TITLE
Add M(sym) macro and map data disassembler

### PR DESCRIPTION
**Pull request by nanaian**

### CODE REVIEW

File: src/world/area_kmr/kmr_12/events.c [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
     SI_SET(SI_SAVE_VAR(425), 31),
     SI_CALL(SetSpriteShading, -1),
     SI_CALL(SetCamPerspective, 0, 3, 25, 16, 4096),
     SI_CALL(SetCamBGColor, 0, 0, 0, 0),
     SI_CALL(SetCamEnabled, 0, 1),
-    SI_CALL(MakeNpcs, 0, npc_groups),
-    SI_EXEC_WAIT(make_entities),
-    SI_EXEC(kmr_12_play_music),
-    SI_SET(SI_VAR(0), bind_exits),
+    SI_CALL(MakeNpcs, 0, M(npcGroupList)),
```
Comment (ethteck) [inline src/world/area_kmr/kmr_12/events.c:19]: Did we decide this should be `Npc` even at the beginning, or am I forgetting? So many "npc" naming conversations :p

**Review by ethteck — APPROVED** [review]

File: src/world/area_kmr/kmr_12/events.c (hunk 1 above)
Comment (nanaian) [reply src/world/area_kmr/kmr_12/events.c:19]: IMO PascalCase should be reserved from script API functions, scripts, and typedefs

Comment (ethteck) [reply src/world/area_kmr/kmr_12/events.c:19]: oh right, this is a variable. carry on

**Review by pixel-stuck — APPROVED** [review]

**Review by pixel-stuck — APPROVED** [review]

**Comment by pixel-stuck** [comment]
I did approve this, but the conflict does need to be resolved

**Pull request merged**